debian-live/build.sh

```diff
@@ -30,7 +30,7 @@ function logerror {
 function importEnvVars {
   loginfo "${FUNCNAME[0]}" "Loading 'build.env' environment variables"
   set -a # automatically export all variables
-  source "${WORK_DIR}"/debian-live/build.env
+  source "${WORK_DIR}"/debian-live/build.env && IMAGE_TIMESTAMP=$(date +%Y%m%d%H%M%S)
```

Comment:
please move it into the env vars section [at the top](https://github.com/batchworksde/iksdp_desktop/blob/f22828eae32fd7fbda0a5c812fded6726dcf1d44/debian-live/build.sh#L6)

```shell=bash
declare USE_CASE WORK_DIR BUILD_DIR IMAGE_TIMESTAMP

USE_CASE=${1}
WORK_DIR="$(pwd)"
BUILD_DIR="${HOME}/build"
IMAGE_TIMESTAMP=$(date +%Y%m%d%H%M%S)

export USE_CASE WORK_DIR BUILD_DIR IMAGE_TIMESTAMP
```